rosbag2_storage/package.xml

```diff
@@ -8,12 +8,18 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <depend>pluginlib</depend>
-  <depend>rcutils</depend>
+  <build_depend>pluginlib</build_depend>
```

Comment:
`build_depend`, `exec_depend` and `test_depend` can be combined within `<depend>...</depend>`